src/modules/systemlib/printload.c

```diff
@@ -116,11 +116,35 @@ tstate_name(const tstate_t s)
 void print_load_buffer(uint64_t t, char *buffer, int buffer_length, print_load_callback_f cb, void *user,
 		       struct print_load_s *print_state)
 {
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wformat" // NuttX uses a different printf format
+#pragma GCC diagnostic ignored "-Wformat-extra-args"
+
+	buffer[0] = 0;
```

Comment:
I'd rather remove the empty line here. The fewer lines there are, the easier it fits into a terminal window.